CHANGELOG.md

```diff
@@ -1,6 +1,30 @@
 # Changelog
 
-## [v1.25.0-rc](https://github.com/googleforgames/agones/tree/v1.25.0-rc) (2022-07-26)
+## [v1.25.0](https://github.com/googleforgames/agones/tree/v1.25.0) (2022-08-02)
+
+[Full Changelog](https://github.com/googleforgames/agones/compare/v1.25.0-rc...v1.25.0)
+
+**Implemented enhancements:**
+
+- Support UDP Load Balancers to allow Agones to work with GKE Autopilot mode [\#2687](https://github.com/googleforgames/agones/issues/2687)
```

Comment:
This enhancement wasn't implemented and shouldn't be part of the changelog. I updated the label on the issue but to update this file you will need to regenerate the changelog.